readme.md

```diff
@@ -19,21 +19,21 @@ SFML is a simple, fast, cross-platform and object-oriented multimedia API. It pr
 
 ## Download
 
-You can get the latest official release on [SFML's website](http://www.sfml-dev.org/download.php). You can also get the current development version from the [Git repository](https://github.com/SFML/SFML).
+You can get the latest official release on [SFML's website](https://www.sfml-dev.org/download.php). You can also get the current development version from the [Git repository](httpss://github.com/SFML/SFML).
 
 ## Install
 
-Follow the instructions of the [tutorials](http://www.sfml-dev.org/tutorials/), there is one for each platform/compiler that SFML supports.
+Follow the instructions of the [tutorials](https://www.sfml-dev.org/tutorials/), there is one for each platform/compiler that SFML supports.
 
 ## Learn
 
 There are several places to learn SFML:
 
-  * The [official tutorials](http://www.sfml-dev.org/tutorials/)
-  * The [online API documentation](http://www.sfml-dev.org/documentation/)
-  * The [community wiki](https://github.com/SFML/SFML/wiki/)
-  * The [community forum](http://en.sfml-dev.org/forums/) ([French](http://fr.sfml-dev.org/forums/))
+  * The [official tutorials](https://www.sfml-dev.org/tutorials/)
+  * The [online API documentation](https://www.sfml-dev.org/documentation/)
+  * The [community wiki](httpss://github.com/SFML/SFML/wiki/)
```

Comment:
Is this the super secure protocol? 😉

Reply:
Yep http super secure is the new telnet!